pkg/cmd/repo/list/http.go

```diff
@@ -208,10 +204,15 @@ func searchQuery(owner string, filter FilterOptions) string {
 			Fork:     fork,
 			Is:       []string{filter.Visibility},
 			Language: filter.Language,
-			Org:      owner,
 			Topic:    []string{filter.Topic},
 		},
 	}
 
+	if owner == "" {
+		q.Qualifiers.User = "@me"
+	} else {
+		q.Qualifiers.Org = owner
```

Comment:
In ES, `user:` and `org:` qualifiers seem largely interchangeable, except (as we've learned) that `org:@me` is not supported. Could we just switch to `user:{owner}` across the board and drop the `Org` qualifier across other commands? That should also fix `gh search ... --owner @me` commands, which are now broken for the same reason